region/lowernorfair/east.json

```diff
@@ -2759,17 +2759,44 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
-                    {"resetRoom":{
-                      "nodes": [1, 4],
-                      "nodesToAvoid": [3]
-                    }},
+                    {"or": [
+                      {"obstaclesNotCleared": ["A"]},
+                      {"heatFrames": 180}
+                    ]},
                     {"heatFrames": 175}
                   ],
-                  "note": "This longer runway's not usable if the shot blocks are destroyed."
+                  "note": "The shot blocks will respawn if destroyed, but resetting the room is faster than waiting."
                 }
               ],
               "usableComingIn": false,
               "openEnd": 0
+            },
+            {
+              "name": "Runway with Blocks Broken While Entering - Red Kihunter Shaft Left Door (to Amphitheatre)",
+              "length": 12,
+              "strats": [
+                {
+                  "name": "Single Missile Beam Shot",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "canPrepareForNextRoom",
+                    "canAutoCancelWeapon",
+                    {"or": [
+                      {"ammo": {"type": "Missile","count": 1}},
+                      "canUseGrapple"
+                    ]},
+                    {"heatFrames": 60}
+                  ],
+                  "note": [
+                    "Enter the room aiming diagonally down, with a missile set to auto-deselect.",
```

Comment:
The super puts your other shots on cooldown so that won't work.
I assumed grapple would work likes missiles and didn't try it without autocancel, so that's neat.
Higher run speed sure is tricky.  Pretty much are trying to buffer the shot before you regain complete control.  Hopefully it's fine at really high run speeds, but by then you can get blue without the runway.